iRecruit-ui/src/components/Profile/Profile.jsx

```diff
@@ -0,0 +1,175 @@
+import React, { useState, useEffect } from "react";
+import './Profile.css';
+
+function Profile() {
+  const [photo, setPhoto] = useState(null);
+  const [bio, setBio] = useState("");
+  const [education, setEducation] = useState("");
+  const [currentInstitution, setCurrentInstitution] = useState("");
+  const [skills, setSkills] = useState("");
+  const [experience, setExperience] = useState("");
+  const [linkedinUrl, setLinkedinUrl] = useState("");
+  //all of these can be updated, hence the need for state hooks
+  const [profileUpdated, setProfileUpdated] = useState(false);
+
+  useEffect(() => {
+    // Retrieve the profile information from localStorage when the component mounts
+    const storedProfile = localStorage.getItem("profile");
+    if (storedProfile) {
+      const { photo, bio, education, currentInstitution, skills, experience, linkedinUrl } = JSON.parse(storedProfile);
+      setPhoto(photo);
+      setBio(bio);
+      setEducation(education);
+      setCurrentInstitution(currentInstitution);
+      setSkills(skills);
+      setExperience(experience);
+      setLinkedinUrl(linkedinUrl);
+    }
+  }, []);
+
+  const handlePhotoChange = (event) => {
+    const file = event.target.files[0];
+    if (file) {
+      const reader = new FileReader();
+      reader.onloadend = () => {
+        setPhoto({
```

Comment:
Not necessary to fix this here, but what does this do if we upload invalid files? Like a text file or something?